Source/WebCore/html/OffscreenCanvas.h

```diff
@@ -191,8 +194,18 @@ class OffscreenCanvas final : public ActiveDOMObject, public RefCounted<Offscree
     mutable bool m_hasCreatedImageBuffer { false };
     bool m_detached { false };
     bool m_hasScheduledCommit { false };
+
+    mutable std::unique_ptr<CSSParserContext> m_cssParserContext;
 };
 
+inline const CSSParserContext& OffscreenCanvas::cssParserContext() const
```

Comment:
Given our use of LTO on most platforms, not sure this needs to be in the header.

Reply:
Done.